client/my-sites/earn/memberships/style.scss

```diff
@@ -20,6 +20,22 @@
 	width: auto;
 }
 
+.memberships__module-content {
+	max-height: 560px;
+	overflow-y: scroll;
+}
+
+.memberships__module-content::-webkit-scrollbar {
```

Comment:
I am not sure about what those 2 rules are about. I would keep the default browser ones...
Also it did not seem to change anything in my Chrome...